WordPress/WordPressTest/BloggingPromptsServiceTests.swift

```diff
@@ -196,8 +170,7 @@ final class BloggingPromptsServiceTests: CoreDataTestCase {
         let expectedPromptIDs = Set(testPrompts.map(\.promptID))
 
         // insert existing prompts.
-        let date = try XCTUnwrap(Self.dateFormatter.date(from: "2022-05-03"))
-        makeBloggingPrompt(siteID: siteID, promptID: 1000, date: date)
+        try makeBloggingPrompt(siteID: siteID, promptID: 1000, date: testPrompts.last!.dateString)
```

Comment:
This test needs to assert that there are never two blogging prompts created for the same calendar day.

It previously used dates for this, but we can simplify it – by comparing something like `2022-05-03` instead of `2022-05-03T12:34:56.000Z`, there's much less opportunity for errors to sneak in. By adjusting the codebase to use a `dateString` for comparison, the logic is quite a bit more robust against issues.

Tiny improvement – we care that the date of the last prompt matches the one in the test data – by referencing it directly we can ensure they don't get out of sync.